scripts/reset-test-db.sh

```diff
@@ -24,17 +24,15 @@
 
 DB_CACHE_DIR="common/test/db_cache"
 
-# Ensure the test database exists.
-echo "CREATE DATABASE IF NOT EXISTS edxtest;" | mysql -u root
-
```

Comment:
@jzoldak I think this line is in the correct place. It deletes and recreates the database. And then in the "if" case lines 34-36 will recreate the tables and loads the initial data from the cache and the lines 39-40 will do the rest of the migrations.

Yes, I have tested all three cases.